### ISSUE TITLE
Scrum Updates May 29

### ISSUE THREAD

**Issue by nawa-ton** [Zephyr0402/PetPal#1]

**Comment by nawa-ton** [comment]
-- CS ID: d1d2b --
On the past iteration, I collaborated with the team on Project Progress #1, as well as prepared sketches for homepage and posting animal page. There is no issue/challenge for me during this time. 

For this coming iteration, I will work with the team to finalize the basic user flow and database design. Then, we should be able to split up the tasks and start working for the next deliverables.

**Comment by runzw** [comment]
-- CS ID: d1d2b --
On the past iteration, I worked on Project Progress [#1](https://github.com/Zephyr0402/PetPal/issues/1). I wrote one of the three requirements and broke it down into specific tasks. No challenge encountered during this.

For this coming iteration, I will be working on database design, which consists of ER Diagram design and relational schema design.

**Comment by LiJu21** [comment]
-- CS ID: o0l6t --
In the past iteration, I helped in coming up with a general project idea and different requirements our web app might meet. The most challenging part for me is to determine and prioritize requirements according to the time limitation. For the coming iteration, I will work on researching some potential APIs to implement map-related functionalities.

**Comment by Zephyr0402** [comment]
-- CS ID: k6j0n --
In the past iteration, I worked with group members to brainstorm the project ideas and finish the tasks on Project Progress #1. I have faced the challenge of analyzing each proposed functionality's implementation difficulty and deciding whether the functionality should be placed in the minimum, standard, or stretch requirement. For the coming iteration, I will reassign the functionalities discussed in workshop 2 to standard requirements, work with the team to come up with the database design and user flow design.
